pubspec.yaml

```diff
@@ -7,7 +7,8 @@ dependencies:
   async: "^1.13.0"
   collection: "^1.5.0"
   http_parser: ">=0.0.1 <4.0.0"
-  path: ">=0.9.0 <2.0.0"
+  mime: ">=0.9.0 < 1.0.0"
```

Comment:
`^0.9.0`

With this version constraint, `mime` could release a totally breaking `0.10.0` release and we'd still declare compatibility with it.